docs/source/release-notes/2021-1-2-Eos.md

```diff
@@ -0,0 +1,72 @@
+# 2021.1.2 Eos Release Notes
+
+This version of Vircadia is an update to 2021.1.1. It is compatible with the 2020.1.0 Eos protocol. This version was released on May 30th, 2021.
+
+## Native Interface & Domain Server (Codename Athena)
+
+The numbers at the end of each item are the PR numbers in the Vircadia [repo](https://github.com/vircadia/vircadia).
+
+### Enhancements
+
+- [**osx**][**windows**] Hash Windows and Mac machine fingerprints. [#1126](https://github.com/vircadia/vircadia/pull/1126)
+- [**domain-server**] The domain server should see Packet ID=1 as a (broken) attempt to connect to the domain server. [#692](https://github.com/vircadia/vircadia/pull/692)
+-  Create app: Copy-paste position and rotation. [#1213](https://github.com/vircadia/vircadia/pull/1213)
+-  Add Doxygen tool for compiling C++ documentation. [#1212, #1220](https://github.com/vircadia/vircadia/pull/1212)
+-  Include Vircadia Web app as a Git submodule. [#1202](https://github.com/vircadia/vircadia/pull/1202)
+-  ReadyPlayerMe blendshapes support. [#1183](https://github.com/vircadia/vircadia/pull/1183)
+-  Update to Qt 5.15.2. [#938](https://github.com/vircadia/vircadia/pull/938)
```

Comment:
It might make sense to mention that this fixes a pretty bad bug on Linux AMD: https://github.com/vircadia/vircadia/issues/801